src/coreclr/build.cmd

```diff
@@ -6,6 +6,7 @@ setlocal EnableDelayedExpansion EnableExtensions
 set "__MsgPrefix=BUILD: "
 
 echo %__MsgPrefix%Starting Build at %TIME%
+echo %__MsgPrefix%WARNING: This build script is deprecated and will be deleted soon. Use the root build script to build CoreCLR. If you want to build the CoreCLR runtime without using MSBuild, use the build-native.cmd script.
```

Comment:
Please file an issue that tracks the deletion of the sub-directory build scripts and add a link above this line. This way we broadcast our intent here and give contributors a place to discuss the issue.